src/query.ts

```diff
@@ -244,77 +252,32 @@ export class Query {
    */
   public setParameters(parameters: SearchParameters): void {
     this.params = Object.assign({}, this.params, this._hydrate(parameters));
-  }
-
-  /**
-   * Sets an exclusion structure to the current context
-   *
-   * @param  {Object}        filters            The exclusion filter to add
-   *
-   */
-  public setExclusions(filters: Facet): void {
-    this.setFilters(filters, 'exclude');
-  }
-
-  /**
-   * Remove a sorting field from the parameters
-   *
-   * @param   {String}    field   The field we want to remove the ordering
-   *
-   */
-  public removeSorting(field: string): void {
-    if (!this.params.sort) {
-      return;
+    if ('sort' in this.params) {
+      this.addSort(this.params['sort']);
```

Comment:
Remove `sort` key from `this.params` after importing sort parameters.

Reply:
done